src/drivers/uploadthing.ts

```diff
@@ -0,0 +1,90 @@
+import { defineDriver } from "./utils";
+import { ofetch, $Fetch } from "ofetch";
+import { UTApi } from "uploadthing/server";
+
+export interface UploadThingOptions {
+  apiKey: string;
+}
+
+export default defineDriver<UploadThingOptions>((opts) => {
+  let client: UTApi;
+  let utFetch: $Fetch;
+
+  const getClient = () => {
+    return (client ??= new UTApi({
+      apiKey: opts.apiKey,
+      fetch: ofetch.native,
+    }));
+  };
+
+  const getUTFetch = () => {
+    return (utFetch ??= ofetch.create({
+      baseURL: "https://uploadthing.com/api",
```

Comment:
Would it makes sense to support an optional option for baseURL? (mainly to allow unit testing)